### PR TITLE
fix: line break cause textareaItem word count corrupt for iOS （#1265)

### PR DESCRIPTION
**Pull request by Roxyhuang**
First of all, thank you for your contribution! :-)

Please makes sure that these checkboxes are checked before submitting your PR, thank you!

* [x] Make sure that you follow antd's [code convention](https://github.com/ant-design/ant-design/wiki/Code-convention-for-antd).
* [x] Run `npm run lint` and fix those errors before submitting in order to keep consistent code style.
* [x] Rebase before creating a PR to keep commit history clear.
* [x] Add some descriptions and refer relative issues for you PR.

Extra checklist:

**if** *isBugFix* **:**

  * [x] Make sure that you add at least one unit test for the bug which you had fixed.

**elif** *isNewFeature* **:**

  * [ ] Update API docs for the component.
  * [ ] Update/Add demo to demonstrate new feature.
  * [ ] Update TypeScript definition for the component.
  * [ ] Add unit tests for the feature.

---
This change is [Reviewable](https://reviewable.io/reviews/ant-design/ant-design-mobile/2825)

### CODE REVIEW

File: components/textarea-item/index.tsx [hunk 1]
```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import TouchFeedback from 'rmc-feedback';
 import { TextAreaItemPropsType } from './PropsType';
 import { Omit } from '../_util/types';
+import { IS_IOS_NOT_DEV_TOOLS } from '../_util/exenv';
```
[the remark this answers is not in the document]
Comment (Roxyhuang) [reply components/textarea-item/index.tsx]: 桌面devtools模拟ios会有问题，这个针对真机的一个bug

[the remark this answers is not in the document]
Comment (Roxyhuang) [reply components/textarea-item/index.tsx]: #1265 里我有提及原因，可以使用真机测试一下

[the remark this answers is not in the document]
Comment (Roxyhuang) [reply components/textarea-item/index.tsx]: 可能描述的不清晰，我补充一下，该问题只有在iOS真机下maxlength会出现问题，使用浏览器的devtools并不会出现，因此需要做特殊处理判断不是devtools模拟的，否则devtools模拟会出问题

[the remark this answers is not in the document]
Comment (doxiaodong) [reply components/textarea-item/index.tsx]: @ziluo  看一下有没有必要加这一个。影响范围是 dev-tools

[the remark this answers is not in the document]
Comment (doxiaodong) [reply components/textarea-item/index.tsx]: @ziluo 你看看需不需要加上这个变量，`dev-tools` 需不需要照顾到

[the remark this answers is not in the document]
Comment (ziluo) [reply components/textarea-item/index.tsx]: 暂不考虑dev-tools

[the remark this answers is not in the document]
Comment (Roxyhuang) [reply components/textarea-item/index.tsx]: @doxiaodong 需要我close之后，修改后再提PR吗？ 如果不判断再dev-tools的iOS的情况下，整个maxLength的限制会失效哦

[the remark this answers is not in the document]
Comment (doxiaodong) [reply components/textarea-item/index.tsx]: @Roxyhuang 你直接改成 IsIOS 呀

[the remark this answers is not in the document]
Comment (Roxyhuang) [reply components/textarea-item/index.tsx]: @doxiaodong 可以了，麻烦review

[the remark this answers is not in the document]
Comment (Roxyhuang) [reply components/textarea-item/index.tsx]: @doxiaodong  可以了，麻烦review一下，谢谢

**Pull request merged**